src/views/index_view.js

```diff
@@ -18,6 +18,7 @@ export default BaseView.extend({
   initialize () {
     // listen for chrome key events
     this.listenTo(webChannel, 'navigational-key', this.dispatchKeypress);
+    window.app.on('adjust-height', debounce(this.adjustHeight, 25));
```

Comment:
If you `import app from 'ampersand-app'` at the top of this file, then you don't need a `window` reference, you can just do `app.on(...`.